### ISSUE TITLE
Saved presets in Transaction Filter and saved column formats

### ISSUE THREAD

**Issue by JuanJabo1GH** [moneymanagerex/moneymanagerex#4796]
Saved presets in Transaction Filter and saved column formats

- The latest versions: mmex-1.5.15-win32 and mmex-1.5.16-win32.
- They are deleting or do not maintain configurations from previous versions, in the case of: Pre-settings saved in Transaction Filter and saved column formats, among others.
- You have to recreate them, something that did not happen or did not happen with the changes in previous versions.
- In short, it seems to eliminate or does not recognize any previous configuration established by the user.

![image](https://user-images.githubusercontent.com/89281363/177819466-c85592d7-569e-43ab-8675-3d5ebd7139c0.png)
![image](https://user-images.githubusercontent.com/89281363/177819819-00bfdb93-5bf4-4487-bba2-96706a03895b.png)

**Comment by renato-mmex** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/4623

**Comment by JuanJabo1GH** [comment]
Hello, please do not consider this problem closed because even in the latest version: mmex-1.5.18 this problem continues, please indicate a solution, they reported that in version: mmex-1.5.17 it had been resolved and that is not real or it was not solved, thanks.
